### DISCUSSION TITLE
RECORD PROBLEM y211ga_0.2.0 IFUS x2 speed video

### DISCUSSION THREAD

**Discussion by Giuserver** [roleoroleo/yi-hack-Allwinner-v2#368] [Q&A]
Hello!
i've noticed when i riproduce the video example: /tmp/sd/record/2022Y02M17D17H/39M00S60.mp4
in my computer the video speed it's x2 and audio speed it's x1
So, the video end after 30 seconds and audio end at 60 seconds (1 minute correctly).
WHY?

I've 4 IFUS cameras with y211ga_0.2.0 (both problem for all)
any idea?

Thanks
Giuseppe

**Comment by roleoroleo** [comment]
If you download the video and play the video locally, does it work?

**Reply by Giuserver** [reply]
> If you download the video and play the video locally, does it work?

I play the video locally.. on my computer and the problem persist. I try also whit my phone, download this file for example /tmp/sd/record/2022Y02M22D10H/21M00S60.mp4 same problem. The video speed x2 (end at 30 sec) audio x1 (end 60 second regularly)

Thanks

**Reply by roleoroleo** [reply]
With or without the hack?

**Reply by Giuserver** [reply]
> With or without the hack?

with hack..
(without hack i can see video recorded through app yi home or kami app, and works)

**Reply by roleoroleo** [reply]
Maybe a problem with the timezone.
Try to remove the timezone setting in the web gui.

**Reply by Giuserver** [reply]
> Maybe a problem with the timezone.
> Try to remove the timezone setting in the web gui.

yes, now works. Thanks

**Reply by roleoroleo** [reply]
Could you test a new beta that should fix this problem?
If yes, I can send it asap.

**Reply by Giuserver** [reply]
> 

yes, where i can find beta hack?

**Reply by roleoroleo** [reply]
I will send it asap in this discussion.

**Reply by roleoroleo** [reply]
Here it is:
[y211ga_0.2.0.tar.gz](https://github.com/roleoroleo/yi-hack-Allwinner-v2/files/8134778/y211ga_0.2.0.tar.gz)

**Reply by roleoroleo** [reply]
Try to set the timezone both in the app and in the web gui.
Please check:
- if the speed is ok
- if date/time reported by the web gui is ok
- if you can play the videos in the Events page (web gui)

**Reply by Giuserver** [reply]
> Try to set the timezone both in the app and in the web gui. Please check:
> 
> * if the speed is ok
> * if date/time reported by the web gui is ok
> * if you can play the videos in the Events page (web gui)

I try :
1) UTC 0 (universal)  (web page and app)
OK DIRECTORY in web page (UTC 0)
SPEED (x2 Video) (x1 Audio) 
OK PLAY VIDEOS from web page

2) UTC +1 (Rome) (web page and app)
NO CORRECT DIRECTORY in web page (still UTC 0, also after reboot and delete old videos)
SPEED (x2 Video) (x1 Audio)
OK PLAY VIDEOS from web page

3) No TZ in web page (UTC 0 in app)
OK DIRECTORY in web page (UTC 0)
SPEED (x2 Video) (x1 Audio) 
OK PLAY VIDEOS from web page


(From YI app play videos seems slowed down, but timing it's ok)
